xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -105,6 +113,23 @@
   private final Map<URI, CacheResourceConfigurationParser> resourceXmlParsers = new HashMap<URI, CacheResourceConfigurationParser>();
   private final ConfigType config;
 
+  static String replaceProperties(String originalValue, final Properties properties) {
```

Comment:
Bleagh, testing seam.  If we have to expose a seam here I'd rather expose the caller of this method.  Then it might potentially be useful in other contexts?